Sources/DistributedActors/_OrderedSet+Extensions.swift

```diff
@@ -0,0 +1,29 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Distributed Actors open source project
+//
+// Copyright (c) 2018-2022 Apple Inc. and the Swift Distributed Actors project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.md for the list of Swift Distributed Actors project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import OrderedCollections
+
+extension OrderedSet {
+    // FIXME(collections): implemented for real in https://github.com/apple/swift-collections/pull/159
+    @inlinable
+    internal func _filter(
```

Comment:
missing operation in OrderedSet; upstreaming better impl: https://github.com/apple/swift-collections/pull/159

Reply:
I think the `uncheckedUniqueElements` initializer would generally give better performance here:

```swift
let members = self.filter(isIncluded)
return OrderedSet(uncheckedUniqueElements: members)
```

This prevents multiple rehashings of the result while it is being constructed. Copying the contents twice would still be slower than what the package can do, but reducing hash operations would still be a considerable boost!

Reply:
Nice, thanks for the hint!